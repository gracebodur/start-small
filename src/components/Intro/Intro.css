.Proj-intro {
    display: flex;
    align-items: center;
    text-align: center;
    color: #222831;
    padding: 20px;
    padding-top: 25%;
}

.Proj-name {
    font-size: 50pt;
    text-transform: uppercase;
}

ul {
    list-style-type: none;
    text-align: center;
    letter-spacing: 0.4ch;
    font-size: var(--fs-h3);
    text-transform: capitalize;
    font-weight: 600;
}

.Proj-demo {
    display: flex;
    justify-content: center;
    align-items: center;
    padding-top: 10%;
    font-size: 20pt;
    font-weight: var(--fw-bold);
    letter-spacing: var(--spacing-extra-small);
    color: var(--clr-dark);
}

@media only screen 
  and (min-device-width: 320px) 
  and (max-device-width: 568px)
  and (-webkit-min-device-pixel-ratio: 2) {
    .Proj-intro {
        margin-top: 10px;
        margin-right: 5px;
     }

    .Proj-name {
         font-size: 25pt;
     }
     
     ul {
         font-size: 15pt;
     }
     
     .Proj-demo {
         font-size: 15pt;
     }
 }
 

/* galaxy */
@media only screen 
  and (min-device-width: 360px) 
  and (max-device-width: 667px) 
  and (-webkit-min-device-pixel-ratio: 2) { 
    .Proj-intro {
        align-items: center;
        margin-top: 2%;
        margin-right: 2%;
     }

     .Proj-name {
         font-size: 23pt;
     }
     
     ul {
         font-size: 13pt;
     }
     
     .Proj-demo {
         font-size: 13pt;
     }
 }


/* iphone6/7/8 */
@media only screen 
  and (min-device-width: 375px) 
  and (max-device-width: 667px) 
  and (-webkit-min-device-pixel-ratio: 2) {
      .Proj-intro {
       align-items: center;
    }

    .Proj-name {
        font-size: 25pt;
    }
    
    ul {
        font-size: 15pt;
    }
    
    .Proj-demo {
        font-size: 15pt;
    }
}

/*iphonex */
@media only screen 
  and (min-device-width: 375px) 
  and (max-device-width: 812px) 
  and (-webkit-min-device-pixel-ratio: 3) {
    .Proj-intro {
        align-items: center;
    }

    .Proj-name {
        font-size: 30pt;
    }
    
    ul {
        font-size: 15pt;
    }
    
    .Proj-demo {
        font-size: 15pt;
    }
   
}

/* ipad */
@media only screen 
  and (min-device-width: 768px) 
  and (max-device-width: 1024px) 
  and (-webkit-min-device-pixel-ratio: 2) {
    .Proj-name {
        font-size: 40pt;
    }
    
    ul {
        font-size: 20pt;
    }
    
    .Proj-demo {
        font-size: 18pt;
    }
  
}

/* ipadpro */
@media only screen 
  and (min-device-width: 834px) 
  and (max-device-width: 1112px)
  and (-webkit-min-device-pixel-ratio: 2) {
    .Proj-name {
        font-size: 50pt;
    }
    
    ul {
        font-size: 25pt;
    }
    
    .Proj-demo {
        font-size: 23pt;
    }
  
}

