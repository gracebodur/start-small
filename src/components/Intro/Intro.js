import React from 'react'
import './Intro.css'

function Intro() {
    return (
        <div>
            <header className="App-header">
                <h1>Start small</h1>
                <h3>Make a big difference!</h3>
            </header>
            <section>
                <ul>
                    <li>Pick a classroom project for your friends to support.</li>
                    <li>See your impact!</li>
                </ul>
            </section>
        </div>
    )
}

export default Intro