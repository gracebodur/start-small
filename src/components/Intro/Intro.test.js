import React from "react";
import toJson from "enzyme-to-json";
import Intro from "./Intro";

describe(`Intro component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a Intro by default", () => {
    const wrapper = <Intro />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the Intro given props", () => {
    const wrapper = <Intro {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
