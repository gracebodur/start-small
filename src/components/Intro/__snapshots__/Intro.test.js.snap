// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro component renders a Intro by default 1`] = `undefined`;

exports[`Intro component renders the Intro given props 1`] = `undefined`;
